pmp-core/src/main/java/dk/nykredit/pmp/core/commit/ParameterRevert.java

```diff
@@ -0,0 +1,34 @@
+package dk.nykredit.pmp.core.commit;
+
+import dk.nykredit.pmp.core.audit_log.ChangeType;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ParameterRevert extends ParameterChange {
```

Comment:
TODO: We need to add a deserializer for this later, so we can actually receive parameter reverts